eZ/Publish/Core/REST/Server/Input/Parser/Criterion/LogicalOperator.php

```diff
@@ -37,6 +37,19 @@ public function parse(array $data, ParsingDispatcher $parsingDispatcher)
      */
     protected function getFlattenedCriteriaData(array $criteriaByType)
     {
+        if ($this->isZeroBasedArray($criteriaByType)) {
+            $oldFormat = $criteriaByType;
+            $criteriaByType = $this->normalizeCriteriaByType($criteriaByType);
+            @trigger_error(
+                sprintf(
+                    'REST View: Passing criteria as a list of objects to a logical operator is deprecated and will cause Bad Request error in eZ Platform 3.0. Instead of "%s" provide "%s"',
```

Comment:
This line is too long and can smuggle a potential typo that may be hidden from the end of the line

Reply:
> This line is too long and can smuggle a potential typo that may be hidden from the end of the line

Fixed via fdf2aea.